x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_details.tsx

```diff
@@ -245,7 +245,9 @@ export const EndpointDetails = memo(
           title: i18n.translate('xpack.securitySolution.endpoint.details.hostname', {
             defaultMessage: 'Hostname',
           }),
-          description: <EuiText>{details.host.hostname}</EuiText>,
+          description: (
+            <EuiText style={{ wordBreak: 'break-all' }}>{details.host.hostname}</EuiText>
```

Comment:
do you think we could use `eui-textBreakAll` class instead of inline css?
https://elastic.github.io/eui/#/utilities/css-utility-classes

Reply:
Thanks, @patrykkopycinski I'll give that a try.

Reply:
done

Reply:
It does work on firefox.